### PR TITLE
Vale: Fix all the remaining errors in the Updating section

### PR DESCRIPTION
**Pull request by AnnaTomanek**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Fix all the remaining errors in the Updating section reported by Vale. Closes cksource/ckeditor5-internal#3641.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/updating/changelog.md [hunk 1]
```diff
@@ -9,10 +9,12 @@ toc-limit: 1
...
+This is the CKEditor&nbsp;5 changelog guide. Here you will find information about the most important changes introduced in the release, new features, and bug fixes.
 
-You can find additional information about each release in the blog posts linked at the start of each entry.
+Information about both major and minor breaking changes is available, too, if the release introduces them. You can read more about breaking changes in CKEditor&nbsp;5 in the [Versioning policy](https://ckeditor.com/docs/ckeditor5/latest/support/versioning-policy.html#major-and-minor-breaking-changes) guide.
 
-This guide only provides the changelog information for the latest 3 releases of CKEditor&nbsp;5. For older releases, refer to the [release notes on GitHub](https://github.com/ckeditor/ckeditor5/releases).
+You can find more information about each release in the blog posts linked at the start of each entry.
```
Comment (godai78) [inline docs/updating/changelog.md:16]: Actually, I don't think we link these anymore.

**Review by godai78 — APPROVED** [review]
LGTM

File: docs/updating/changelog.md (hunk 1 above)
Comment (AnnaTomanek) [reply docs/updating/changelog.md:16]: Looking at [https://ckeditor.com/docs/ckeditor5/latest/updating/guides/changelog.html,](https://ckeditor.com/docs/ckeditor5/latest/updating/guides/changelog.html,) I don't see the blog posts linked anywhere. So what's the desired state?

Comment (godai78) [reply docs/updating/changelog.md:16]: We don't mention posting blogposts at the current time. This may subject to change back in the future.

**Pull request merged**